simulator/kernel/mars/mars_kernel.cpp

```diff
@@ -14,6 +14,8 @@
 #include <unordered_map>
 #include <vector>
 
+const uint64 TRAP_VECTOR_BASE_ADDRESS = 0x8'000'0000;
```

Comment:
This variable should have `static constexpr` attributes, and its type should be Addr.

Reply:
I don't think it's a good scope for the variable used in a single function. Could you please move it into the function?